test/test_prototype_builtin_datasets.py

```diff
@@ -44,6 +44,14 @@ def test_coverage():
 
 @pytest.mark.filterwarnings("error")
 class TestCommon:
+
+    _TRANSFORMS = transforms.Compose(
```

Comment:
Please add a comment that these transforms are a proxy for a preprocessing pipeline that we use to have an "end-to-end" test.